app/code/community/Divante/VueStorefrontExternalCheckout/etc/config.xml

```diff
@@ -39,5 +51,13 @@
             </checkout_onepage_controller_success_action>
         </events>
     </global>
+    <default>
+        <vuestorefrontexternalcheckout>
+            <vuestorefrontexternalcheckout_group>
+                <exclude_route><![CDATA[checkout/onepage]]></exclude_route>
+                <externalcheckout_url>order-success</externalcheckout_url>
```

Comment:
I would still prefer when this is empty as it other wise would be a breaking change for stores as it was previews an empty string so you were not redirected to vsf success page.